src/software/simulated_tests/simulated_test_simulator.h

```diff
@@ -0,0 +1,92 @@
+#pragma once
+
+#include <memory>
+#include <vector>
+
+#include "software/proto/messages_robocup_ssl_wrapper.pb.h"
+#include "software/time/timestamp.h"
+#include "software/world/ball_state.h"
+#include "software/world/field.h"
+#include "software/world/robot_state.h"
+
+/**
+ * This class wraps Simulator for the SimulatedTestFixture
+ */
+class SimulatedTestSimulator
```

Comment:
I understand the problem this is solving, but don't think the name conveys this, its almost misleading

I don't have a better suggestion, but maybe we can come up with something w/ `Wrapper` or `Adapter` (and maybe `Nanopb`, but maybe not)

Reply:
changed to SerializedToNanoPbSimulatorAdapter